examples/shard-server.config.example

```diff
@@ -87,6 +87,16 @@ instances {
       # than a catastrophic failure.
       action_blacklist_prefix: "ActionBlacklist"
 
+      # A redis key prefix for all blacklisted invocations, suffixed
+      # with the tool invocation id. Requests on behalf of an invocation
+      # which is blacklisted should be rejected where it is identified via
+      # its RequestMetadata
+      # To meet API standards, a request which matches this condition
+      # receives a transient UNAVAILABLE response, which, in bazel's
+      # case, can induce a fallback to non-remote recovery, rather
+      # than a catastrophic failure.
+      invocation_blacklist_prefix: "InvocationBlacklist"
```

Comment:
because "unavailable" is the closest thing we have to "go away" response?  
In this context, I'm also guessing its also wrong to respond with "failed condition" or something similar.

Reply:
This is the response that is compatible with --remote_local_fallback. This reminds me that we need to have a 'fallback-friendly' option and some more definitive responses here for non-fallback compatible buildfarms.

Reply:
thx. I need to read up on all the possible responses that are expected for the various remote apis